src/tui.rs

```diff
@@ -258,37 +232,45 @@ impl Widget for &mut App {
         // Create a space for header, todo list and the footer.
         let vertical = Layout::vertical([
             Constraint::Length(2),
-            Constraint::Min(0),
+            Constraint::Fill(1),
```

Comment:
This is functionally the same but better shows the intent, IMO. Pretty needless change otherwise 😅